lib/Doctrine/DBAL/Schema/Schema.php

```diff
@@ -384,6 +415,50 @@ public function dropSequence($sequenceName)
         return $this;
     }
 
+    public function hasView(string $viewName): bool
+    {
+        $viewName = $this->getFullQualifiedAssetName($viewName);
+
+        return isset($this->_views[$viewName]);
+    }
+
+    /**
+     * @throws SchemaException
+     */
+    public function getView(string $viewName) : View
+    {
+        $viewName = $this->getFullQualifiedAssetName($viewName);
+        if (! isset($this->_views[$viewName])) {
+            throw SchemaException::viewDoesNotExist($viewName);
+        }
+
+        return $this->_views[$viewName];
+    }
+
+    /**
+     * @return View[]
+     */
+    public function getViews() : array
+    {
+        return $this->_views;
+    }
+
+    public function createView(string $viewName, string $sql) : View
+    {
+        $view = new View($viewName, $sql);
+        $this->_addView($view);
+
+        return $view;
+    }
+
+    public function dropView(string $viewName) : self
```

Comment:
Would additional `@return $this` make sense?

Reply:
Not sure. But since `Schema` is not final, I think it makes sense.